bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -122,6 +123,22 @@ trait UtilBijections {
       override def apply(pool: FuturePool) = new TwitterExecutionContext(pool)
       override def invert(context: ExecutionContext) = new ScalaFuturePool(context)
     }
+
+  object Owned {
```

Comment:
something like:

```scala
case class Owned(bytes: Array[Byte]) extends AnyVal
case class Shared(bytes: Array[Byte]) extends AnyVal
```
then you can do `buf.as[Shared].bytes` if we want to go that way. I kind of like usage of types for this kind of decoration.